Assets/Scripts/UI/PickupPup/DogWorld/DogWorld.cs

```diff
@@ -10,22 +10,20 @@ public class DogWorld : MonoBehaviourExtended
 {	
     [SerializeField]
     PPScene room;
+    [SerializeField]
+    UIButton deselectArea;
 
-    DogWorldSlot[] dogsSlots;
+    static DogWorldSlot selectedDogSlot = null;
```

Comment:
I added it so we could enforce that there would only be one selected dog. I didn't know if there would ever be a case where we would end up with two instances without realizing it that would point to different things, so I avoided that scenario. Personally not attached to it though, if you want it removed.